pkgs/by-name/es/espresso/package.nix

```
@@ -0,0 +1,43 @@
+{ lib, fetchFromGitHub, cmake, stdenv, ninja, nix-update-script }:
+stdenv.mkDerivation rec {
+  pname = "espresso";
+  version = "2.4";
+  src = fetchFromGitHub {
+    owner = "chipsalliance";
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "sha256-z5By57VbmIt4sgRgvECnLbZklnDDWUA6fyvWVyXUzsI=";
+  };
+
+  nativeBuildInputs = [ cmake ninja ];
```

```suggestion
  nativeBuildInputs = [ cmake ];
```

`ninja` is only faster on incremental builds.